### PR TITLE
[BXMSDOC-4566] Misha edits September 2019

### PR DESCRIPTION
**Pull request by mramendi**
Signed-off-by: Mikhail Ramendik <mramendi@redhat.com>
@errantepiphany @spolti please review APB version.
@sterobin I woukd appreciate a peer review too

After reviews are cleared I'l create a 7.5.x PR too (just a cherry-pick)

### CODE REVIEW

**Comment by mramendi** [comment]
@ruromero I am not at all sure that I have the right environment to run this, though. I would much prefer if my PR was reviewed and merged and then someone else ran the script (separately on 7.5.x branch too, I'll be cherry-picking there)

**Comment by spolti** [comment]
Misha, all you need to do is do _./prepare.sh_.

**Comment by ruromero** [comment]
> Misha, all you need to do is do _./prepare.sh_.

The script depends on the `apb` client. I added a commit to this PR

**Comment by spolti** [comment]
the script only does a perl command, why does it depends on apb client?

**Comment by ruromero** [comment]
> the script only does a perl command, why does it depends on apb client?

Ignore me, I forgot the script was changed. You are right

File: apb.yml [hunk 1]
```diff
@@ -100,7 +100,7 @@ _apb_businesscentral_hostname: &_apb_businesscentral_hostname
 _apb_controller_global_discovery: &_apb_controller_global_discovery
```
Comment (sterobin) [inline apb.yml]: @mramendi Only comment is that it's "KIE Server" or "KIE Servers" capitalized. If it's already "KIE server" everywhere else, then I guess whatever. Just fyi.

Also, as a heads-up, we will NOT be unifying Process Server/Decision Server to KIE Server this release as we had hoped. It's fine as it is here though and in file names, just leave everything however it is normally. The prod team was not able to address prod impact for 7.5,  so we bumped to 7.6. Last you and I spoke I said I'd let you know if we would do so for any impact in the templates, but said to press on in the absence of news. So I'm confirming that there is now change on server terminology this release.

Comment (mramendi) [reply apb.yml]: The lines I changed already had "KIE server" and I really don't want to push on alignment, given this is getting deprecated anyway.

**Comment by mramendi** [comment]
@ruromero @spolti so exactly what do I do next about this PR? I have cherry-picked to 7.5.x and ran prepare.sh there and it is a different PR now.

**Comment by spolti** [comment]
@mramendi cherry pick your commit apply to the new branch for 7.5.x then execute the prepare.sh script, I think this should be fine.

**Comment by errantepiphany** [comment]
Looks like the changes are okay and the prepare.sh script was run. I've approved it, but have re-requested reviews from @spolti and @ruromero before it gets merged.

**Pull request merged**